src/GeneralStateTestsFiller/stCreateTest/CreateResultsFiller.yml

```diff
@@ -0,0 +1,171 @@
+                     ; write the contract to constructor memory 
+                     (lll 
+                         (mstore 0xFF 0xFF)
+                         contractCode
+                     )
+              )
+              constructorCode
+          )
+
+          ; Write the contract to our own memory (as the 0xcc...cc contract)
+          [contractLength] (lll 
+             (mstore 0xFF 0xFF)
+             contractCode
+           )
+
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; Actually create the contract
+          ;
+          (if (= @creation 1)
+            [addr1] (create 0 constructorCode @constructorLength)
+            [addr1] (create2 0 constructorCode @constructorLength 0x5a17)
+          )
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; RETURNDATASIZE should be zero
+          ;
+
+          [[0x10]] (returndatasize)
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; EXTCODE should match what we returned in the constructor
+          ;
+
+          [extLength] (extcodesize @addr1)
+          (extcodecopy @addr1 extCode 0 @extLength)
+
+          [[0x11]] (- @contractLength @extLength)
+          [[0x12]] (- @contractCode   @extCode)
+
+          ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+          ; CALL the created contract
+
+          (if (= @callType 1)
+             [callRet] (call 0xFFFF @addr1 0 0 0 0 0)
+             NOP
+          )
+
+          (if (= @callType 2)
+             [callRet] (callcode 0xFFFF @addr1 0 0 0 0 0)
+             NOP
+          )
+
+          (if (= @callType 3)
+             [callRet] (delegatecall 0xFFFF @addr1 0 0 0 0)
+             NOP
+          )
+
+          (if (= @callType 4)
+             [callRet] (staticcall 0xFFFF @addr1 0 0 0 0)
+             NOP
+          )
+
+          [[0x13]] (- @callRet 1)
+          [[0x14]] (returndatasize)  
+
+        }   ; end of LLL code
+      nonce: '0'
+      storage: {}
+
+
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '0x0ba1a9ce0ba1a9ce'
+      code: '0x'
+      nonce: '0'
+      storage: {}
+
+
+  transaction:
+    data:
+    # MSB
+    #
+    # Create type:    01 - CREATE     02 - CREATE2
+    # Call   type:    01 - CALL    02 - CALLCODE    03 - DELEGATECALL  04 - STATICCALL
+    #
+    # LSB  
+    - 0x0101
+    - 0x0102
+    - 0x0103
+    - 0x0104
+    - 0x0201
+    - 0x0202
+    - 0x0203
+    - 0x0204
+    gasLimit:
+    - 0x100000000
+    gasPrice: '1'
+    nonce: '0'
+    to: cccccccccccccccccccccccccccccccccccccccc
+    secretKey: "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
+    value:
+    - 0
+
+  expect:
+    - indexes:
+        data: 
+        - !!int -1
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - '>=Istanbul'
+      result:
+        cccccccccccccccccccccccccccccccccccccccc:
+          storage: {}
+
+            # Zero if all goes well 
```

Comment:
put all the storage to 12 for example. so that if all goes well. 12 is overwritten with 0.
in case tx fails the storage would be 0 too